src/shared/components/query/QueryStore.ts

```diff
@@ -1,1734 +1,1740 @@
-            }
-        }
-
-        if (this.isQueryLimitReached) {
-            return "Please limit your queries to 100 genes or fewer.";
-        }
-
-        if (this.genes.result.suggestions.length)
-            return "Please edit the gene symbols.";
-    }
-
-    private readonly dict_molecularAlterationType_filenameSuffix: {[K in MolecularProfile['molecularAlterationType']]?: string} = {
-        "MUTATION_EXTENDED": 'mutations',
-        "COPY_NUMBER_ALTERATION": 'cna',
-        "MRNA_EXPRESSION": 'mrna',
-        "METHYLATION": 'methylation',
-        "METHYLATION_BINARY": 'methylation',
-        "PROTEIN_LEVEL": 'rppa',
-    };
-
-    @computed get downloadDataFilename() {
-        let study = (this.selectableSelectedStudyIds.length === 1 && this.treeData.map_studyId_cancerStudy.get(this.selectableSelectedStudyIds[0]));
-        let profile = this.dict_molecularProfileId_molecularProfile[this.selectedProfileIds[0] as string];
-
-        if (!this.forDownloadTab || !study || !profile)
-            return 'cbioportal-data.txt';
-
-        let suffix = this.dict_molecularAlterationType_filenameSuffix[profile.molecularAlterationType] || profile.molecularAlterationType.toLowerCase();
-        return `cbioportal-${study.studyId}-${suffix}.txt`;
-    }
-
-    readonly asyncUrlParams = remoteData({
-        await: () => [this.asyncCustomCaseSet],
-        invoke: async () => currentQueryParams(this)
-    });
-
-    ////////////////////////////////////////////////////////////////////////////////
-    // ACTIONS
-    ////////////////////////////////////////////////////////////////////////////////
-
-    /**
-     * This is used to prevent selections from being cleared automatically when new data is downloaded.
-     */
-    public initiallySelected = {
-        profileIds: false,
-        sampleListId: false
-    };
-
-
-    sanitizeQueryParams(str?: string) {
-        return str ? decodeURIComponent(str).replace(/\+/g, "\n") : "";
-    }
-
-    @action setParamsFromUrl(url: string | { [k: string]: Partial<CancerStudyQueryUrlParams> }) {
-        let params: Partial<CancerStudyQueryUrlParams>;
-        if (typeof url === 'string') {
-            let urlParts = URL.parse(url, true);
-            params = urlParts.query as Partial<CancerStudyQueryUrlParams>;
-        } else {
-            params = url; // already an object
-        }
-
-        let profileIds = [
-            params.genetic_profile_ids_PROFILE_MUTATION_EXTENDED,
-            params.genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION,
-            params.genetic_profile_ids_PROFILE_MRNA_EXPRESSION,
-            params.genetic_profile_ids_PROFILE_METHYLATION,
-            params.genetic_profile_ids_PROFILE_PROTEIN_EXPRESSION,
-            params.genetic_profile_ids_PROFILE_GENESET_SCORE,
-            params.genetic_profile_ids_PROFILE_GENERIC_ASSAY
-        ];
-
-        let queriedStudies = params.cancer_study_list ? params.cancer_study_list.split(",") : (params.
```

Comment:
Our style guide says no underscore prefixes to indicate intended scope, so I renamed this from `_genesetQuery`